google/cloud/grpc_options_test.cc

```diff
@@ -107,10 +107,47 @@ TEST(GrpcChannelArguments, MakeChannelArguments) {
   expected.SetString("foo", "bar");
   expected.SetString("baz", "quux");
   expected.SetUserAgentPrefix("user_agent");
+  expected.SetInt(GRPC_ARG_KEEPALIVE_TIME_MS, 86400000);
+  expected.SetInt(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 60000);
```

Comment:
Ahhhh... I think I gave you bad advice. I now see that these two lines are necessary, because we are checking for equality of the `grpc::ChannelArguments` objects. Let's call this the defaults test where no user option is supplied.

I have a nit though. If I stumbled on the code it would not be obvious what `86400000` means. Can we instead say: `std::chrono::milliseconds(std::chrono::hours(24)).count()`? and do the same for `60000`?